app/forms/idv/in_person/address_form.rb

```diff
@@ -35,11 +35,13 @@ def submit(params)
       attr_reader :capture_secondary_id_enabled
       alias_method :capture_secondary_id_enabled?, :capture_secondary_id_enabled
 
-      def consume_params(params)
-        params.each do |key, value|
-          raise_invalid_address_parameter_error(key) unless ATTRIBUTES.include?(key.to_sym)
-          send("#{key}=", value)
-        end
+      def set_params(params)
+        @state = params[:state]
```

Comment:
big fan of unrolling here! we may not even seen the separate method, could just inline this!